MAP/src/components/Map/Map.js

```diff
@@ -540,16 +540,15 @@ export class Map extends React.Component {
 
   render() {
    const {isCountrySearchVisible} = this.props;
-   
-    
+   const {isZoomVisible} = this.props;
```

Comment:
you can merge it to one variable 
`const {isCountrySearchVisible,  isZoomVisible} =  this.props`

Reply:
it is just small comments